privacyidea/lib/apps.py

```diff
@@ -153,6 +153,7 @@ def create_google_authenticator_url(key=None, user=None,
         hash_algo = ""
 
     if tokentype.lower() == "totp":
+        tokentype = "totp"
```

Comment:
I meant something like
`tokentype = tokentype.lower()`
at the beginning of the function. So what ever is passed to the function as a tokentype is converted to lower case.